include/spdlog/details/registry.h

```diff
@@ -68,6 +70,11 @@ class SPDLOG_API registry {
         periodic_flusher_ = details::make_unique<periodic_worker>(clbk, interval);
     }
 
+    std::unique_ptr<periodic_worker> &get_flusher() {
+        std::lock_guard<std::mutex> lock(flusher_mutex_);
+        return periodic_flusher_;
+    }
```

Comment:
I am not sure this makes much sense here. This local lock object will not limit concurrent access to the object via the returned reference.

Reply:
But it does protect the actual instance used by the registry. Anyway we are not using it, so fine.